crates/core/tedge_agent/Cargo.toml

```diff
@@ -24,6 +24,7 @@ http-body = { workspace = true }
 http-body-util = { workspace = true }
 hyper = { workspace = true, features = ["full"] }
 log = { workspace = true }
+mqtt_channel = { workspace = true }
```

Comment:
This won't be needed once MQTT actor is capable of establishing dynamic connections as well.

Reply:
Resolved by bf84271